scripts/docker/run_project.sh

```diff
@@ -21,7 +21,7 @@ slug=$1
 rounds=$2
 timeout=$3
 
-iDFlakiesVersion=1.2.0-SNAPSHOT
+iDFlakiesVersion=2.0.0-SNAPSHOT
 
 # Setup prolog stuff
 cd "/home/$SCRIPT_USERNAME/$TOOL_REPO/scripts/"
```

Comment:
See the couple of commands below for usage of modify-project.sh

Reply:
Should we pass in groupID and artifact_id to this script as user inputs as well? Or set those to defaults and only pass in the version?